packages/contracts/src/contracts/current/protocol/AssetProxy/interfaces/IAssetData.sol

```diff
@@ -0,0 +1,33 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.23;
+
+interface IAssetData {
```

Comment:
Where is this being used?

Reply:
I wanted to use it as 
```
bytes4 constant PROXY_ID = IAssetData.ERC20Token.selector
```
and 
```
const tokenAsset = IAssetData.ERC20Token.getABIEncodedTransactionData(tokenAddress);
```

Unfortunately neither work.

Reply:
I added documentation explaining the purpose.